### PR TITLE
Add snapshot test

### PR DESCRIPTION
**Pull request by KyleAMathews**
Based on this Google Doc I created with text covering all the normal formatting options.

This will give us a good base to fix pulling in, strong, italics, strike through, underline, etc. into the markdown. I also noticed that unordered lists are created as ordered lists.

### CODE REVIEW

File: src/convert-google-document-to-json.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+function getText(element) {
+  const text = cleanText(element.textRun.content)
+  const {link} = element.textRun.textStyle
+
+  return link ? `[${text}](${link.url})` : text
+}
+
+module.exports = data => {
+  // console.log(`${process.cwd()}data.json`)
```
Comment (xuopled) [inline src/convert-google-document-to-json.js]: Could you remove these comments please?

Comment (KyleAMathews) [reply src/convert-google-document-to-json.js]: gone!

Comment (xuopled) [reply src/convert-google-document-to-json.js]: Thank you

**Comment by xuopled** [comment]
I just published `v1.3.0` with your commit

**Pull request merged**